macos/FluentUITestApp/TestControlsViewController.swift

```diff
@@ -17,9 +17,9 @@ fileprivate struct Constants {
 class TestControlsViewController: NSViewController, NSTableViewDelegate, NSTableViewDataSource {
 	private let controls: [(title: String, type: NSViewController.Type)] = [
 							("FluentUI-macOS (placeholder)", TestPlaceholderViewController.self),
```

Comment:
Unrelated to this change, should we remove this placeholder view controller?

Reply:
I vote we keep it but replace the logo with the FluentUI Logo, and maybe have a Link (hehe) to the website